src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -30,14 +35,16 @@ export const transformIdentify: IdxStepTransformer = ({ formBag, widgetProps, tr
     uischema.elements as UISchemaElement[],
   ) as FieldElement;
   if (identifierElement) {
-    // add username/identifier from config if provided
-    if (username) {
-      data.identifier = username;
-    // TODO: OKTA-508744 - to use rememberMe in features once Default values are added widgetProps.
-    // (i.e. rememberMe is default = true in v2)
-    } else if (features?.rememberMe !== false && features?.rememberMyUsernameOnOIE) {
-      const usernameCookie = getUsernameCookie();
-      data.identifier = usernameCookie;
+    if (!isClientTransaction) {
```

Comment:
nit: Can combine the two nested if's 
```suggestion
    if (identifierElement && !isClientTransaction) {
```